joint_limits:
  j0:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  j1:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  j2:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  j3:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  j4:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0